scripts/README.md

```diff
@@ -17,7 +17,34 @@
   * Commit changes and push to origin
   * Open github to submit the PR
 
-## Update sitemap
+### Extracting texts from js code:
+Texts that used in js code would automatically be extracted during the translation update process (3rd step above: Updating the source file).
 
-* List of paths to include in `sitemap.xml` is here: [sitemap.js](sitemap.js)
+In order to make sure that the extractor code won't miss any string to push to translations, please:
```

Comment:
perhaps clearer?
`In order to make sure that no strings are missed by the extractor code while pushing to translations, please`